root/dataframe/CMakeLists.txt

```diff
@@ -224,3 +224,4 @@ ROOTTEST_ADD_TEST(test_progressiveCSV
 
 # need the '+' to autogenerate required dictionaries with ACLiC
 ROOTTEST_ADD_TEST(test_nested_rvec_snapshot MACRO test_nested_rvec_snapshot.C+)
+ROOTTEST_ADD_TEST(test_snapshot_copyaddresses MACRO test_snapshot_copyaddresses.C+)
```

Comment:
Why do you ACLiC this, is it needed?

Reply:
it's just a copy-paste from the line above. apparently at some point Danilo or I thought we needed to ACLiC the test to be able to perform I/O of custom types defined in the test source (see the comment two lines above), but that's not really the case isn't it?

Reply:
Yes, it is needed for that. (i.e. we technically do not (fully) support I/O of interpreted classes.

Reply:
D'oh of course, thanks!